endpointmanager/pkg/endpointmanager/postgresql/fhirendpointinfostore_test.go

```diff
@@ -226,7 +261,7 @@ func Test_PersistFHIREndpointInfo(t *testing.T) {
 	}
 
 	// get the first update and check its value
-	rows = store.DB.QueryRow("SELECT http_response, capability_statement FROM fhir_endpoints_info_history WHERE id=$1 AND operation='U' ORDER BY entered_at ASC LIMIT 1;", endpointInfo1.ID)
+	rows = store.DB.QueryRow("SELECT http_response, capability_statement FROM fhir_endpoints_info_history, fhir_endpoints_metadata WHERE fhir_endpoints_info_history.metadata_id = fhir_endpoints_metadata.id AND operation='U' AND fhir_endpoints_info_history.id=$1 ORDER BY fhir_endpoints_info_history.entered_at ASC, fhir_endpoints_info_history.entered_at ASC LIMIT 1;", endpointInfo1.ID)
```

Comment:
Why is the `entered_at` field ordered twice here and on line 277?

Reply:
Whoops typo, fixed!